app/assets/stylesheets/src/shared/_components.scss

```diff
@@ -112,13 +112,15 @@
   .selectize-input {
     border-radius: 4px !important;
     vertical-align: middle; // Fix the height of the component.
-
-    input::placeholder {
-      color: $gray-600;
-    }
+    background-color: var(--tblr-white);
+    border-color: var(--tblr-border-color);
 
     &.focus {
-      border-color: tint-color($blue, 50%) !important;
+      border-color: #83b7e8 !important;
+
+      box-shadow:
+        $box-shadow-input,
+        0 0 0 0.25rem rgba(var(--tblr-primary-rgb), 0.25);
```

Comment:
Was it for something specific? It seems to work without it

Reply:
just to be sure that it will change if tabler update `--tblr-primary-rgb`

Reply:
will remove

Reply:
though, it is not the same
tblr used 6, 111, 209; while selectize 13, 110, 253
![image](https://github.com/user-attachments/assets/64163488-3d65-4d4e-a9f8-562f7e9072ae)